### ISSUE TITLE
can´t remove items from grid

### ISSUE THREAD

**Issue by oisjferf** [refinedmods/refinedstorage#3616]
### Describe the bug

So I fixed the bug before, I removed all useless armor/tools that I got from my farms and now the grid shows the items. But suddenly for some reason even though I haven't really changed anything, I can't take anything out anymore? Here is a video of this glitch. (I am trying to take things out the whole time ( [shift] left and rightclick ) ). https://youtu.be/cUD4nEDFvFg

and this is still the ATM8 modpack, relevant mods: extra disks, refined storage, cable tiers, extra storage, rsinfinitybooster.

### How can we reproduce this bug or crash?

1. Having many farms that generate/produce items
2. Having external storages ( I am using the functional storage mod )
3. I don`t really know ho to reproduce this. It was everything normal and for no reason at all I couldn´t take out any items and this weird "non stackable" thing happened ( You can see in this video with redstone, wither skulls etc. )

### What Minecraft version is this happening on?

Minecraft 1.19.2

### What Forge version is this happening on?

43.2.14

### What Refined Storage version is this happening on?

1.11.7

**Comment by DeeJayhX** [comment]
Ahh yes, this common pitfall. I recently had to figure this out myself. 9/10 times, it's because you attached an external storage to a "machine" of sorts. You're using the machine's every-changing output slots as a "storage" and that's what is causing your problem. Either use an importer on the machine, or if you are trying to only have "so much" of the item, pipe the items into a chest/barrel/other storage block and use the External Storage on that instead.

tl:dr; basically do not connect "External Storage" to ANYTHING except for storage blocks (chest/barrel/crate/etc)

I am not a developer on this project, just a fellow user. Ta!

**Comment by oisjferf** [comment]
> Ahh yes, this common pitfall. I recently had to figure this out myself. 9/10 times, it's because you attached an external storage to a "machine" of sorts. You're using the machine's every-changing output slots as a "storage" and that's what is causing your problem. Either use an importer on the machine, or if you are trying to only have "so much" of the item, pipe the items into a chest/barrel/other storage block and use the External Storage on that instead.
> 
> tl:dr; basically do not connect "External Storage" to ANYTHING except for storage blocks (chest/barrel/crate/etc)
> 
> I am not a developer on this project, just a fellow user. Ta!

I connected it to my storage controller which controls drawers, which controls my mob farm. Is that also a "storage block"?

**Comment by BDTheNerdyMedic** [comment]
> I connected it to my storage controller which controls drawers, which controls my mob farm. Is that also a "storage block"?

The storage controller is fine, but make sure you do not have any external storage attached to ender chests or tanks as these will kill your grid as soon as the inventory changes.

**Comment by ArtichaTM** [comment]
Met same problem with connection between **External Storage -> Interface**. If place interface first, and forgot to set _Extract-only_ instantly, interface became simple storage. Funny part is, if you switch _External Storage_ to _Extract-Only_ when there are items in interface, they become bugged. Displays in grid, displays everywhere, but you can't pick it, export or craft something. By the way, this method described in [refined storage wiki](https://refinedmods.com/refined-storage/autocrafting/keeping-fluids-in-stock.html#nav-keeping-fluids-in-stock)

Fix is simple, destroy interface and place again, but now don't forget to turn _extract-only_ before connecting to network

I tried this myself in new creative world, and picking items from interface actually removes them from grid, **BUT NOT ALL**. One item stayed in grid, but in interface there's no such item. 
<details><summary>Screenshots</summary>




Structure
![Structure](https://github.com/refinedmods/refinedstorage/assets/46131671/abc854d6-41c1-42ba-8345-e8d1a0fc43e0)

Interface
![Interface](https://github.com/refinedmods/refinedstorage/assets/46131671/2f07b576-211a-468d-b04c-6ea444f33ae2)

Grid
![Grid](https://github.com/refinedmods/refinedstorage/assets/46131671/5788a248-2a9e-4e26-ae18-6edd47d3347d)
</details>

May be this bug caused by switching _external storage_ from _Insert and Extract_ to _Extract-only_, but I'm too lazy to check this myself

**Comment by raoulvdberge** [comment]
Duplicate of #2568
